### PR TITLE
Translation Appendices and Security page

### PR DESCRIPTION
**Pull request by octisfall**

### CODE REVIEW

File: ru/security.rst [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
-    button to directly propose your changes.
-
-    You can refer to the english version in the select top menu to have
-    information about this page's topic.
+CakePHP предоставляет инструменты для обеспечения безопасности
+вашего приложения. Следующие разделы охватывают эти инструменты:
 
 .. toctree::
-    :maxdepth: 1
+:maxdepth: 1
```
Comment (markstory) [inline ru/security.rst]: This needs to be indented 4 spaces.

**Comment by markstory** [comment]
Tjanks!

**Comment by octisfall** [comment]
@markstory can i ask one question? Um.. are you planning to pay for translations?

**Comment by markstory** [comment]
@octisfall No. Everything we do in CakePHP is volunteer based. Even core contributors are volunteers.

**Comment by octisfall** [comment]
Wow.. Then i want to just say thank you for your work. You're a great man and i believe in what you do.

**Comment by markstory** [comment]
You're welcome and thank you for helping make CakePHP better 👏

**Pull request merged**